### PR TITLE
DP-25147: Remove locations-old

### PR DESCRIPTION
**Pull request by arthurbaghdas**
**Description:**
Explain the technical implementation of the work done.

**Jira:** (Skip unless you are MA staff)
DP-25147

**To Test:**
- [ ] Add steps to test this feature

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -255,7 +255,7 @@ jobs:
       - run: php scripts/directory-yaml-parse.php
       # https://architecture.lullabot.com/adr/20211201-use-strict-types-in-php-code/
       # New php files must use strict types.
-      - run: scripts/phpcs-new-files
+      # - run: scripts/phpcs-new-files
```
Comment (tom-fleming) [inline .circleci/config.yml]: @arthurbaghdas Is this meant to be commented out?

File: scripts/phpcs-new-files [hunk 2]
```diff
@@ -10,4 +10,5 @@ git diff-tree \
             -z \
             --name-only \
             --no-commit-id \
+            --diff-filter=ACMRTUXB \
```
Comment (tom-fleming) [inline scripts/phpcs-new-files:13]: @arthurbaghdas Is this a change that we still want in this file? Seems unrelated to the ticket.

Comment (arthurbaghdas) [reply scripts/phpcs-new-files:13]: I tried to fix deleted file issue related to the strict types check without success and left here to revisit later

File: .circleci/config.yml (hunk 1 above)
Comment (arthurbaghdas) [reply .circleci/config.yml]: Yes, this was communicated with Moshe, and we decided to revisit this in a separate ticket which is already in a spec state

**Comment by tom-fleming** [comment]
LGTM
